src/pages/tasks/NewTaskDetailsPage.js

```diff
@@ -20,15 +20,33 @@ const propTypes = {
     /** Beta features list */
     betas: PropTypes.arrayOf(PropTypes.string),
 
+    /** Task title and description data */
+    task: PropTypes.shape({
+        title: PropTypes.string,
+        description: PropTypes.string,
+    }),
+
     ...withLocalizePropTypes,
 };
 
 const defaultProps = {
     betas: [],
+    task: {},
 };
 
 const NewTaskPage = (props) => {
     const inputRef = useRef();
+    const [taskTitle, setTaskTitle] = useState(props.task.title);
+    const [taskDescription, setTaskDescription] = useState(props.task.description);
+
+    useEffect(() => {
+        if (taskTitle !== props.task.title) {
+            setTaskTitle(props.task.title);
+        }
+        if (taskDescription !== props.task.description) {
+            setTaskDescription(props.task.description);
+        }
+    }, [props.task, taskTitle, taskDescription]);
```

Comment:
I think have taskTitle, taskDescription here causing the below issue

I am unable to update the title value when coming back to new task page.


https://github.com/Expensify/App/assets/46707890/8936546c-ea40-4145-9346-70e07ba86e33

Reply:
I have removed the comparison from the `useEffect` which was causing this. The `useEffect` will now run only once when the component mounts, since the only dependency is `props.task` which is coming from Onyx.